__docs__/wonder-blocks-tabs/navigation-tabs-variants.stories.tsx

```diff
@@ -211,20 +211,52 @@ export default meta;
 export const Default: Story = {};
 
 export const Hover: Story = {
-    parameters: {pseudo: {hover: true}},
+    parameters: {
+        pseudo: {hover: true},
+        chromatic: {
+            // TODO(WB-1917): Temporarily disabled since snapshots with
+            // NavigationTabs and pseudo states are flaky. We still get state
+            // style coverage in NavigationTabItem AllVariants stories
+            disable: true,
```

Comment:
Maybe this should be `disableSnapshot` instead? I'm seeing the snapshots still being included here: https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=67e71a51097d57f7d1951deb